README.md

```diff
@@ -130,14 +130,32 @@ Click the "Code" button above, click the "Codespaces" tab, and then click the "+
 
 ## Running the pipeline
 Before running the pipeline, make sure you are in the same folder as the `create-publication.py` file by entering the following command into the terminal:
+
 `cd RAP_example_pipeline_python`
 
 To run the pipeline, enter the following command into the terminal:
+
 `python create_publication.py`
 
-## Project structure
+## Running the tests
+There are two sets of tests in this structure: 
+
+* **Unit tests**: these test functions in isolation to ensure they do what you expect them to.
+* **Back tests**: when you refactor a pipeline or re-create it entirely, it's a good idea to compare the results of the old process (often referred to as the "ground truth") to the results of the new pipeline. This is what the back tests do. Here, the back tests will first check if the output files exist in the data_out folder, and if not, it will run the pipeline and create these files so that it can compare them to the ground truth files (stored in the `tests/backtests/ground_truth/` folder). Note that you don't need to commit your ground truth files to your repo (for example if they are very large or contain sensitive data).
```

Comment:
```suggestion
* **[Back tests](https://nhsdigital.github.io/rap-community-of-practice/training_resources/python/backtesting/)**: when you refactor a pipeline or re-create it entirely, it's a good idea to compare the results of the old process (often referred to as the "ground truth") to the results of the new pipeline. This is what the back tests do. Here, the back tests will first check if the output files exist in the data_out folder, and if not, it will run the pipeline and create these files so that it can compare them to the ground truth files (stored in the `tests/backtests/ground_truth/` folder). Note that you don't need to commit your ground truth files to your repo (for example if they are very large or contain sensitive data).
```